addons/web/static/src/js/views/abstract_view.js

```diff
@@ -378,6 +384,11 @@ var AbstractView = Factory.extend({
             withSearchBar: inline ? false : this.withSearchBar,
             withSearchPanel: this.withSearchPanel,
         };
+        // FIXME: is this necessary?
```

Comment:
I think it was for _onOpenView in dashboard

Reply:
to check

Reply:
it is the case

Reply:
you can remove the FIXME then :D